python/gink/tests/test_directory.py

```diff
@@ -153,7 +153,7 @@ def test_reset():
     for store in [LmdbStore(), MemoryStore()]:
         # TODO: implement reset in memory store
         with store:
-            # pylint: disable=unsupported-assignment-operation, unsupported-role-test
+            # pylint: disable=unsupported-assignment-operation, unsupported-group-test
```

Comment:
Is this supposed to change from "unsupported-role-test" to "unsupported-group-test"?
I don't know anything about PyLint, so maybe it is, just figured this could be an error.

Reply:
I don't think that line does anything anymore so I'll just remove it.